cloudfoundry/managers/v3appdeployers/standard_strategy_v3.go

```diff
@@ -49,7 +49,7 @@ func (s Standard) Deploy(appDeploy AppDeploy) (AppDeployResponse, error) {
 			return err
 		}
 
-		jobURL, _, err := s.client.DeleteApplication(app.GUID)
+		jobURL, _, _ := s.client.DeleteApplication(app.GUID)
```

Comment:
Would this never fail? Is there a way to warn the user in case it still does?

Reply:
Fixed.